saleor/product/models.py

```diff
@@ -392,6 +404,16 @@ def __str__(self):
     def quantity_available(self):
         return max(self.quantity - self.quantity_allocated, 0)
 
+    def get_costs_data(self):
```

Comment:
Could this live outside of the model class? We try to prevent models from implementing business logic.

Reply:
CC @mociepka @elwoodxblues

Reply:
I put this into the model to have a simple access to it in templates.
Now i see that this can be splited into two functions get_costs and get_margins. What do you think?